### DISCUSSION TITLE
Make name badge appear on certain components while they're selected (not just on hover)

### DISCUSSION THREAD

**Discussion by benryanwilliams** [GrapesJS/grapesjs#4823] [Q&A]
I've got a set of custom table components (in the example below there's `table`, `for-each tbody`, `tr` and `td` components) that are each selectable by clicking the `td` component then clicking the up arrow on the toolbar, but since the name badge only appears on hovering (and there's no padding between the elements so you can't hover over them), it's very difficult to tell which component is currently selected:

https://user-images.githubusercontent.com/67364267/210473053-a91dc345-ab89-48eb-912e-63c85e6834f5.mov

All I need to be able to do is display the name badge on these components when they are selected (not just when in `hover` state). It would also be helpful to change the position of the name badge inside of the component so that it doesn't interfere with the toolbar:

<img width="1214" alt="Screenshot 2023-01-04 at 09 39 32" src="https://user-images.githubusercontent.com/67364267/210474472-0b506de6-8414-4a31-a8b2-45cbafda072b.png">

I only need this behaviour for the custom table components - I don't want all components to behave this way.

Thanks in advance.

**Comment by artf** [comment]
I think we would need a new option to handle badge visibility (boolean or as a function for custom logic) for selected component as right now it's not a easily customizable behaviour (PRs are welcome).
